cozy/model/track.py

```diff
@@ -35,7 +35,7 @@ def name(self):
         if self._db_object.name:
             return self._db_object.name
 
-        return "{} {}".format(_("Chapter"), self.number)
+        return f"{_('Chapter')} {self.number}"
```

Comment:
Would be good to embed the placeholder into the string, but that's for another PR